### DISCUSSION TITLE
SpeedCrunch issue

### DISCUSSION THREAD

**Discussion by southisup** [ScoopInstaller/Extras#8429] [Q&A] [answered]
I have SpeedCrunch installed and up to date via Scoop Extras. I just discovered that the default for its Radix Character Input Format Setting was `Dot And Comma` rather than the safer and more usual `System Default`. Is this something set by work done in this team, or should I raise an issue over at SpeedCrunch?

e.g. of issue:
```
1,000*1.1
= 1.1

1,000.01*1.1
= 1100.011

**Comment by rashil2000** [comment, marked as answer]
https://github.com/ScoopInstaller/Extras/blob/master/bucket/speedcrunch.json

The manifest does not make any changes to the app configuration

**Comment by southisup** [comment]
Thanks, Issue 1128 created, although development there is not very active.
https://bitbucket.org/heldercorreia/speedcrunch/issues/1128/default-install-radix-character-input
